images/build/Dockerfiles/cli.Dockerfile

```diff
@@ -1,5 +1,3 @@
-ARG DEBIAN_FLAVOR
```

Comment:
Isn't this DEBIAN_FLAVOR used in the actual FROM command?
`FROM buildpack-deps:${DEBIAN_FLAVOR}-curl as main`

Also, i'm not sure that it is redundant, based on https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact ? Maybe I'm not understanding correctly though

Reply:
Yep just realized this is being used in the FROM command. Apologies!